src/data/causes/cats.json

```diff
@@ -0,0 +1,54 @@
+{
+  "path": "cats2",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "#FF6A08",
+        "secondary": "#29BEBA",
+        "primaryContrast": "#15AAA6"
+      }
+    },
+    "causeId": "CA6A5C2uj",
+    "metadata": {
+      "title": "Tab for Cats - Home",
+      "ogTitle": "Help Shelter Cats for Free Online | Tab for Cats ",
+      "url": "/cats/",
+      "ogDescription": "Open new tabs, help shelter cats for free... it’s that easy!",
+      "ogImage": "../../img/cats/1.png",
+      "causeSpecificKeywords": "charity, cat, cats"
+    },
+    "sections": {
+      "landing": {
```

Comment:
modeled so that jed can set the data and query the way he designed it in his component